tests/test_mesh1d_basics.py

```diff
@@ -127,7 +127,7 @@ def test_contacts_compute_single():
     mk.contacts_compute_single(node_mask, polygon, projection_factor)
 
     contacts = mk.contacts_get()
-    sort_contacts_by_mesh2d_indices(contacts)
+    sort_contacts_by_mesh2d_indices(contacts = contacts)
```

Comment:
I hate to be picky, but most times the argument names and values are not separated by spaces around the equal signs: https://stackoverflow.com/questions/8853063/pep-8-why-no-spaces-around-in-keyword-argument-or-a-default-parameter-value refers to PEP8.